integration-templates/kustomer/kustomer-conversations.ts

```diff
@@ -0,0 +1,32 @@
+import type { NangoSync, KustomerConversation } from './models';
+
+export default async function fetchData(nango: NangoSync): Promise<void> {
+    try {
+        const endpoint = '/v1/conversations';
+        const config = {
+            paginate: {
+                type: 'link',
+                link_path_in_response_body: 'links.next',
+                limit_name_in_request: 'pageSize',
+                response_path: 'data',
+                limit: 100
+            }
+        };
+        for await (const conversation of nango.paginate({ ...config, endpoint })) {
+            const mappedConversation: KustomerConversation[] = conversation.map(mapConversation) || [];
+            await nango.batchSave(mappedConversation, 'KustomerConversation');
+        }
+    } catch (error) {
+        throw new Error(`Error in fetchData: ${error}`);
```

Comment:
Error stringify to {} 

```suggestion
        throw new Error(`Error in fetchData`, { cause: error });
```

Reply:
@bodinsamuel just updated my pull request.